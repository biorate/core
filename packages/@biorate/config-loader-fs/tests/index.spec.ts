import { expect } from 'chai';
import { root } from './__mocks__';

describe('@biorate/config-loader-fs', function () {
  before(async () => await root.$run());

  it('config.json', () => expect(root.config.get('root')).to.be.a('boolean').equal(true));

  it('package.json', () => expect(root.config.get('package')).toMatchSnapshot());

  it('config.debug.json', () =>
    expect(root.config.get('debug')).to.be.a('boolean').equal(true));

  it('config.nested.json', () =>
    expect(root.config.get('nested')).to.be.a('boolean').equal(true));

  it('config.tmp.json', () =>
    expect(root.config.get('temp.temp')).to.be.a('boolean').equal(true));
});
