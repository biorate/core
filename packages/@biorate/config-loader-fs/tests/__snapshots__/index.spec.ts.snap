// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@biorate/config-loader-fs package.json 1`] = `
Object {
  "author": "llevkin",
  "description": "Test package.json",
  "keywords": Array [],
  "license": "MIT",
  "name": "file-config-test",
  "version": "0.0.0",
}
`;
